packages/components/src/components/Select/index.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { useCallback } from 'react';
+import styled from 'styled-components';
+import css from '@styled-system/css';
+import VisuallyHidden from '@reach/visually-hidden';
+import { uniqueId } from 'lodash-es';
+import { Text } from '../Text';
+import { Element } from '../Element';
+
+// Svg used for the icon
+const svg = (hover: boolean) =>
+  `"data:image/svg+xml,%3Csvg width='8' height='24' viewBox='0 0 8 24' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M4.00006 17L1 13L7 13L4.00006 17Z' fill='${
+    hover ? 'white' : '%23757575'
```

Comment:
we can't hard code colors 🤷‍♀ Let's replace this with `currentcolor` and style it in the Select component

As to what colors should we use - it should either be the same as text or a muted version, cc @DannyRuchtie

Reply:
You are right, fixed :)